python/ml4ir/base/config/parse_args.py

```diff
@@ -151,6 +151,20 @@ def define_args(self):
             help="Run ID for the current training. Autogenerated if not specified.",
         )
 
+        self.add_argument(
+            "--run_group",
+            type=str,
+            default="general",
+            help="Run group defining a meta grouping to filter different model training runs for best model selection as a post step.",
```

Comment:
_Ullimague: I'm not sure I understand the help without extra context.
Could we namely state: why should we set it, what are the options, where are they stated, what is the outcome of it?_

_lastmansleeping: I'll elaborate it. It is just a name to identify a group of model training runs. run_id is the unique identifier for each model training run. run_group is one level above. You can later use it to group similar runs and select best model._

Reply:
We should say so, maybe in a less abstract way.
Could we just add:
- it is an identifier used to group the different run_id.
- can later use it to group similar runs and select best model.
- where is it output? (basically where can we access it back)?

Reply:
I can add that. You can access it in the experiment tracking data.